autoware_iv_auto_msgs_converter/include/autoware_iv_auto_msgs_converter/autoware_iv_auto_msgs_converter.hpp

```diff
@@ -75,6 +75,26 @@ inline auto convert(const autoware_auto_system_msgs::msg::AutowareState & state)
   return iv_state;
 }
 
+inline auto convert(const autoware_system_msgs::msg::AutowareState & state)
+{
+  autoware_auto_system_msgs::msg::AutowareState auto_state;
+  if (state.state == autoware_system_msgs::msg::AutowareState::INITIALIZING_VEHICLE) {
+    auto_state.state = autoware_auto_system_msgs::msg::AutowareState::INITIALIZING;
+  } else if (state.state == autoware_system_msgs::msg::AutowareState::WAITING_FOR_ROUTE) {
+    auto_state.state = autoware_auto_system_msgs::msg::AutowareState::WAITING_FOR_ROUTE;
+  } else if (state.state == autoware_system_msgs::msg::AutowareState::PLANNING) {
+    auto_state.state = autoware_auto_system_msgs::msg::AutowareState::PLANNING;
+  } else if (state.state == autoware_system_msgs::msg::AutowareState::WAITING_FOR_ENGAGE) {
+    auto_state.state = autoware_auto_system_msgs::msg::AutowareState::WAITING_FOR_ENGAGE;
+  } else if (state.state == autoware_system_msgs::msg::AutowareState::DRIVING) {
+    auto_state.state = autoware_auto_system_msgs::msg::AutowareState::DRIVING;
+  } else if (state.state == autoware_system_msgs::msg::AutowareState::ARRIVAL_GOAL) {
+    auto_state.state = autoware_auto_system_msgs::msg::AutowareState::ARRIVED_GOAL;
+  }
+  auto_state.state = autoware_auto_system_msgs::msg::AutowareState::FINALIZING;
```

Comment:
It looks like this line is always overwriting the state.

Reply:
:innocent: 
https://github.com/tier4/AutowareArchitectureProposal_msgs/pull/12/commits/114f74dc9f267b60dd9c5face73da8e226806e88

Actually I don't use this function, but someone may need this function so let's merge this PR when it's ready.